front/src/modules/conflict/styles/_conflictList.scss

```diff
@@ -76,13 +76,15 @@
     .conflict-type {
       text-align: left;
       flex-grow: 1;
+      white-space: nowrap;
     }
     .conflict-times {
       text-align: right;
       font-size: 0.8rem;
+      white-space: nowrap;
```

Comment:
<img width="434" alt="Capture d’écran 2024-05-30 à 11 00 51" src="https://github.com/OpenRailAssociation/osrd/assets/113104137/e4662a39-bdb6-4cdf-b29e-348a0fc5c86e">

You can add some margin-left here (and will be probably have to handle the overflow when shrinking screen)